modules/ble.md

```diff
@@ -0,0 +1,132 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+
+## <img class="constrain-sm" src="//i.imgur.com/7qR5OZJ.png"> Bluetooth Low Energy
+
+[<i class="fa fa-github"> View source on Github</i>](https://github.com/sandeepmistry/noble)
+
+### Step 1
+
+Make a directory inside your "tessel-code" folder called "ble", change directory into that folder, and initialize a tessel project:
+
+`mkdir ble; cd ble; t2 init`
+
+### Step 2
+</div>
+</div>
+
+<div class="row">
+<div class="large-6 columns">
+
+Plug the BLE module into either of Tessel's USB ports, then plug Tessel into your computer via USB.
+
+</div>
+<div class="large-6 columns">
+
+![](http://i.imgur.com/uifn1p7.jpg)
+
+</div>
+</div>
+
+<div class="row">
+<div class="large-12 columns">
+
+### Step 3
+
+Make sure your Tessel is connected to a wifi network as shown on the [wifi page](/wifi.html). Then gain root access to the Tessel shell:
+
+`t2 root`
+
+This command gives you direct access to the Linux system running on Tessel. We need this in order to power up the USB dongle. Run this command:
+`hciconfig hci0 up`
```

Comment:
`hciconfig` is only for Bluetooth. It should be made to happen automatically when a Bluetooth device is attached.

"power up" is not necessarily the right term. T2 does have the ability to fully cut power to USB devices under software control, but that's separate from enabling a Bluetooth HCI interface.